src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/ConfinedScope.java

```diff
@@ -39,7 +40,6 @@
  */
 final class ConfinedScope extends ResourceScopeImpl {
 
-    private boolean closed; // = false
     private int lockCount = 0;
     private int asyncReleaseCount = 0;
     private final Thread owner;
```

Comment:
Is this `owner` field still needed now that the super class also has it?

Reply:
Forgot to remove - well spotted